frontend/src/components/cart-page/CartItemsSection.tsx

```diff
@@ -0,0 +1,50 @@
+import { useContext } from "react";
+import { useNavigate } from "react-router-dom";
+
+import { ShoppingCartContext } from "../../contexts/shopping-cart-context/shoppingCartContext";
+import { Card, CardContent, Typography } from "@mui/material";
+
+import cartStyles from "./CartStyles";
```

Comment:
Since it's not a component, could you write it as `cartStyles`?

Reply:
Resolved